stock_barcode_alternative/models/stock.py

```diff
@@ -0,0 +1,319 @@
+﻿from odoo import api, fields, models, _
+from odoo import http
+from odoo.http import request
+from odoo.tools import safe_eval as eval
+import traceback
+
+from odoo.modules.registry import Registry
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class BarcodeController(http.Controller):
+
+    @http.route(['/barcode2/web/'], type='http', auth='user')
```

Comment:
Failed smoketest where I tried to access /barcode2/web
Expected behavior: Seeing something similar to what the current module show. 
!Screenshot of current page](https://user-images.githubusercontent.com/72398241/118498571-771d9600-b726-11eb-85bb-6264fad527b0.png)